extensions/parquet/table/src/test/java/io/deephaven/parquet/table/S3ParquetLocalStackTest.java

```diff
@@ -22,8 +23,13 @@ public Builder s3Instructions(Builder builder) {
         return LocalStack.s3Instructions(builder);
     }
 
+    @Override
+    public Map<String, String> s3Properties() {
+        return LocalStack.s3Properties();
+    }
```

Comment:
I wasn't aware of these methods; really, this underlying method is very specific to Apache Iceberg catalog stuff and should be removed to a more appropriate place.

Can you please create 4 static methods in LocalStack and Minio that exposes each of these properties instead and delete `s3Properties`? And then explicitly re-constitute the map in IcebergMinIOTest & IcebergLocalStackTest, ideally using keys from `org.apache.iceberg.aws.AwsClientProperties` and `org.apache.iceberg.aws.s3.S3FileIOProperties`.

(This may become unnecessary in the future if https://github.com/deephaven/deephaven-core/pull/6113 gets merged.)